style.css

```diff
@@ -1034,6 +1158,133 @@ footer img {
     display: flex;
     gap: 40px;
   }
+
+  /* Pop up */
+
+  .pop-up-section {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 1156px;
```

Comment:
I would't recommend having a fixed width for your popups because this will bring you a lot of responsiveness issues 😥 If you have a normal desktop screen size of 1024px, your popup will have rendering issues as seen in the following image 

![image](https://user-images.githubusercontent.com/15807343/162071599-5f6c805f-4048-4bbb-b29b-9f9cd028e74c.png)

It's important that your popup is always shorter than your screen width. You could try using something like `width: 75vw` that way you ensure that it is always 75% of your screen width 🧐 Kindly make the necessary changes so that your popups show properly on mobile and desktop

Reply:
Thank you, done!